backend/category/dao/retreive.py

```diff
@@ -0,0 +1,34 @@
+import sys
+sys.path.append('..')
+from database.connection import *
+from flask import request
+
+
+def retrieve_category(catLevel1,catLevel2,pageNumber,sortKey):
+    '''
+    For given catlevel1 and catlevel2 parameters, the functions fetches the product details from the category and product database.
+    '''
+    res=connectDB()
+    conn=res[0]
+    cur=res[1]
+    if(catLevel2=="women" or catLevel2=="men"):
```

Comment:
Keep it modular, for example the below if-else can be separated in a new function.